packages/server/src/api/todo/social/getNotifications.ts

```diff
@@ -72,6 +71,7 @@ export default function (server: Server) {
         record: JSON.parse(notif.record),
         isRead: notif.createdAt <= user.lastSeenNotifs,
         indexedAt: notif.indexedAt,
+        // @TODO do we need createdAt so that it can be used as a cursor?
```

Comment:
I'd recommend we add an opaque `cursor` property to any paginated record.
```suggestion
        // @TODO do we need createdAt so that it can be used as a cursor?
```

Reply:
yup i like it 👍

Reply:
Awesome, I spun off an issue for it here: https://github.com/bluesky-social/adx/issues/227